README.md

```diff
@@ -1,3 +1,7 @@
+> NOTE: This package has been renamed to angular_components without the "2". 
+> To continue to receive the latest versions update your `pubspec.yaml` to 
+> depend on `angular_components: ^0.5.0-beta`. 
+
```

Comment:
Put a link here to the new package URI on pub.dartlang.org.

And delete the rest of the README.